content/help/releases/2019-004.md

```diff
@@ -10,7 +10,7 @@ menu:
     parent: releases
     weight: 201601
 aliases:
-  - /help/releases/2019-003
+  - /help/releases/2019-004
```

Comment:
This alias can actually be removed entirely. They're essentially redirects, but this one would do nothing.